src/main/java/by/innowise/monolithwebcalculator/domain/mapper/OperationMapper.java

```diff
@@ -3,18 +3,22 @@
 import by.innowise.monolithwebcalculator.domain.operation.Operation;
 import by.innowise.monolithwebcalculator.web.dto.OperationRequestDto;
 import by.innowise.monolithwebcalculator.web.dto.OperationResponseDto;
-import org.springframework.stereotype.Component;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.Mappings;
 
-@Component
-public class OperationMapper {
-    public Operation mapOperationRequestDtoToOperation(OperationRequestDto dto) {
-        Operation op = new Operation();
-        op.setArgOne(dto.getArgOne());
-        op.setArgTwo(dto.getArgTwo());
-        return op;
-    }
-
-    public OperationResponseDto mapOperationToOperationResponseDto(Operation op) {
-        return new OperationResponseDto(op.getArgOne(), op.getArgTwo(), op.getResult(), op.getOperationType());
-    }
+@Mapper(componentModel = "spring")
+public interface OperationMapper {
+    @Mappings({
+            @Mapping(target = "argOne", source = "dto.argOne"),
```

Comment:
Is it necessary to write "dto." before argOne and argTwo? I think mapper will understand without it